prow/github/report/report_test.go

```diff
@@ -205,7 +205,7 @@ func TestParseIssueComment(t *testing.T) {
 			isBot := func(candidate string) bool {
 				return candidate == "k8s-ci-robot"
 			}
-			deletes, entries, update := parseIssueComments(pj, isBot, tc.ics)
+			deletes, entries, update := parseIssueComments([]prowapi.ProwJob{pj}, isBot, tc.ics)
```

Comment:
It would be good to test the new functionality for reporting multiple PJs at once. Please add unit tests for this at some level.

Reply:
will do